tests/generic_relations/tests.py

```diff
@@ -513,6 +514,55 @@ def test_subclasses_with_parent_gen_rel(self):
         TaggedItem.objects.create(content_object=bear, tag='orange')
         self.assertEqual(Carrot.objects.get(tags__tag='orange'), bear)
 
+    def test_generic_inline_formsets_initial_count(self):
+        """
+        Test for initial form count considering save_as_new.
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Tests for" since all tests test things.